.github/actions/high-priority-prs/src/index.js

```diff
@@ -1,9 +1,34 @@
 const fetch = require("./fetch.js")
 const { processData, report } = require("./process-data.js")
+const { filter } = require("./filter.js")
+
+const teams = [
+  {
+    id: `MDQ6VGVhbTI5MzQzMzQ=`,
+    name: `core`,
+    channelId: process.env.SLACK_CORE_CHANNEL_ID,
+  },
+  {
+    id: `MDQ6VGVhbTI5MzI3MjM=`,
```

Comment:
I created a new `learning` team, id `MDQ6VGVhbTMzMDc4MzY=`. Then once this PR is merged we can delete the docs team - this is so there's no gap between the CODEOWNERS file changing and the new learning team being created.